omod/src/main/webapp/resources/htmlforms/003-HMIS_ACP_ARTCard-SummaryPage .xml

```diff
@@ -898,7 +912,7 @@
                                                   <label>
                                                       <span class="required">*</span>
                                                       Pt Clinic #:</label>
-                                                          <patient field="identifier" identifierTypeId="e1731641-30ab-102d-86b0-7a5022ba4115" required="false"/>
+                                                          <patient field="identifier" identifierTypeId="e1731641-30ab-102d-86b0-7a5022ba4115" required="false" id="patientNo"/>
```

Comment:
@dbaluku please use the full name for this ID. i.e, patientNumber and not patientNo

Reply:
Done